src/OrbitGl/LiveFunctionsController.cpp

```diff
@@ -194,31 +194,25 @@ void LiveFunctionsController::OnDeleteButton(uint64_t id) {
   if (id == id_to_select_ && !current_textboxes_.empty()) {
     id_to_select_ = current_textboxes_.begin()->first;
   } else if (current_textboxes_.empty()) {
-    // TODO: Not sure this is a good idea...
     id_to_select_ = 0;
```

Comment:
kInvalidFunctionId?

And, are these ids now function_ids?

Reply:
done.

and yes - they are function ids now.

Reply:
ok, now it is invalid (somehow did not change it in previous ps)